src/components/Nav/LoginForm.jsx

```diff
@@ -103,44 +102,42 @@ export default function LoginForm() {
         <form onSubmit={formSubmit}>
           <ModalHeader toggle={toggleModal}>Login</ModalHeader>
           <ModalBody>
-            <Form>
-              <FormGroup className="mb-2 mr-sm-2 mb-sm-0">
-                <Label htmlFor="username" className="mr-sm-2">
-                  Username
-                </Label>
-                <Input
-                  type="username"
-                  name="username"
-                  id="username"
-                  value={formState.username}
-                  onChange={inputChange}
-                  placeholder="That thing you called yourself"
+            <FormGroup className="mb-2 mr-sm-2 mb-sm-0">
```

Comment:
Reactstrap is weird. As long as this didn't break the formatting and functionality that they baked into Form. I know moving seemingly redundant bits ruined other things I worked on.